src/models/UserMapper.php

```diff
@@ -642,4 +642,15 @@ public function resetPassword($token, $password)
 
         return false;
     }
+
+    public function delete($user_id)
+    {
+        try {
+            $sql = "delete from user where ID = :user_id";
+            $stmt = $this->_db->prepare($sql);
+            return $stmt->execute(array("user_id" => $user_id));
```

Comment:
@heiglandreas and I discussed this about 14 days ago - as it stands, the API and Web2 handle it between them in that just by deleting the user, talks that were linked to that user become 'unlinked' and comments become 'anonymous' in the frontend by virtue of the user not existing.

As such, we suggested it worked as it was, but that would make a potential 'Easy Pick' or 'Hackathon' issue.

Having said that, if you'd rather it got sorted now, I can look at it.